planetaryimage/image.py

```diff
@@ -26,10 +29,25 @@ def open(cls, filename):
         filename : string
             name of file to read as an image file
         """
-        with open(filename, 'rb') as fp:
-            return cls(fp, filename)
+        if filename.endswith('.gz'):
+            try:
+                with gzip.open(filename, 'rb') as fp:
+                    return cls(fp, filename, compression='gz')
+            except AttributeError:
+                raise IOError('GZIP compression not supported until Python 2.6.')
+                return
+        elif filename.endswith('.bz2'):
+            try:
+                with bz2.open(filename, 'rb') as fp:
```

Comment:
We can use [`bz2.BZ2File`](https://docs.python.org/2.6/library/bz2.html#bz2.BZ2File) and [`gzip.GzipFile`](https://docs.python.org/2.6/library/gzip.html#gzip.GzipFile) which will give us identical support back to python2.6. I'm not sure we should support python2.5. Thoughts @godber?